scripts/deploy.sh

```diff
@@ -48,6 +48,21 @@ if [ "${DEPLOY_AIO}" != false ]; then
   # can diagnose the cause of the apt fetch failures.
   export ANSIBLE_PACKAGE="git+https://github.com/rcbops/ansible@v2.3-OSA_SHA-with_apt_errors"
 
+  # RO-4211
+  # Implement debug output for apt so that we can see more information
+  # about whether the 'Acquire-by-hash' feature is being used, and what
+  # might be causing it to fall back to the old style.
+  # This config file should be copied into containers by the lxc_hosts
+  # role.
+  echo 'Debug::Acquire::http "true";' > /etc/apt/apt.conf.d/99debug
```

Comment:
This is not idempotent.  I'm not sure it that is a problem, but I wanted to highlight it just in case.

Reply:
Actually I'm dumb.  That's only 1 `>`